composer.lock

```diff
@@ -4107,6 +4107,54 @@
             ],
             "time": "2023-09-19T05:39:22+00:00"
         },
+        {
+            "name": "psr/clock",
```

Comment:
The addition of `psr/clock` and `symfony/clock` packages looks good and indicates that time-related functionality is being decoupled from system calls, promoting better testability and abstraction. Ensure that this aligns with the project’s PHP version requirements, especially since `symfony/clock` requires PHP >=8.1. Also, confirm that these dependencies are actually used in the source code and not pulled in unintentionally.